.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Wing Development",
+	"image": "mcr.microsoft.com/devcontainers/base:jammy",
+  "features": {
+    "ghcr.io/devcontainers/features/aws-cli:1": {},
+    "ghcr.io/devcontainers/features/terraform:1": {},
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {},
+    "ghcr.io/devcontainers/features/node:1": {
+      "version": 18
+    },
```

Comment:
Yes to the node version but it looks like the feature only allows to specify `node` and `nvm` versions
https://github.com/devcontainers/features/tree/main/src/node

Reply:
having this should take care of both npm and node

Reply:
Yea I think Id like to revisit this later, im going to use the volta feature but that pushes the node and npm installs to post build volta commands since the feature does not have a way to include install during the build scripts. But the trade off is worth it for now to keep pinned versions I think.